src/routes/console/project-[project]/auth/templates/smsLoginTemplate.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+    import SmsTemplate from './smsTemplate.svelte';
+    import LocaleOptions from './localeOptions.svelte';
+    import type { Models } from '@appwrite.io/console';
+    import { page } from '$app/stores';
+    import { loadSmsTemplate } from './+page.svelte';
+    import { baseSmsTemplate, smsTemplate } from './strote';
+    import { addNotification } from '$lib/stores/notifications';
+    import { Id } from '$lib/components';
+
+    export let localeCodes: Models.LocaleCode[];
+
+    const projectId = $page.params.project;
+    let locale = 'en';
+    let loading = false;
+
+    async function onLocaleChange() {
+        const timeout = setTimeout(() => {
+            loading = true;
+        }, 1000);
+        try {
+            const template = await loadSmsTemplate(projectId, 'login', locale);
+            clearTimeout(timeout);
+            smsTemplate.set(template);
+            $baseSmsTemplate = { ...$smsTemplate };
+        } catch (error) {
+            clearTimeout(timeout);
```

Comment:
Instead of doing clear timeout in both places, you can just do it once inside the finally block

Reply:
If you;re using this convention in other files, please fix it there too